interface-definitions/include/firewall/common-rule.xml.i

```diff
@@ -0,0 +1,326 @@
+<!-- include start from firewall/common-rule.xml.i -->
```

Comment:
Any reason the include starts not at the most left column?

Reply:
Fixed